src/site/markdown/server/manage-guice-distributed-james.md

```diff
@@ -337,6 +340,52 @@ to sanitize his mailbox denormalization.
 In order to ensure being offline, stop the traffic on SMTP, JMAP and IMAP ports, for example via re-configuration or 
 firewall rules.
 
+### Mailboxes Counters
+
+Cassandra maintains a per mailbox projection for message count and unseen message count. Failures during the denormalization 
+process will lead to inconsistencies between the counts and the content of `imapUidTable`.
+
+#### How to detect the inconsistencies
+
+Invalid message count/message unseen count can be reported in the Mail User Agent (IMAP & JMAP).
```

Comment:
Invalid **or incorrect** message count/message unseen count can be seen in Mail User Agents (IMAP or JMAP).

Reply:
Invalid values are reported in the logs as warning with the following class `org.apache.james.mailbox.model.MailboxCounters`
and the following message prefix `Invalid mailbox counters`.